src/containers/Root.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Provider } from 'react-redux'
+import { Route } from 'react-router-dom'
+import App from './App'
+
+const Root = ({ store }) => (
+    <Provider store={store}>
+        <div>
+            <Route path="/" component={App} />
```

Comment:
i'd like to avoid common names here

Reply:
do you need jQuery for `bootstrap-react`? ))

Reply:
nice catch